charts/dkube-deployer/templates/config-map.yaml

```diff
@@ -138,8 +138,49 @@ data:
     # E.g: sdb
     STORAGE_CEPH_DISK={{ .Values.optional.storage.cephDisk }}
     [MODELMONITOR]
-    #To enable modelmonitor in dkube. (true / false)
+    # To enable modelmonitor in dkube. (true / false)
     ENABLED={{ .Values.optional.modelmonitor.enabled }}
+
+    [TIMESCALE]
+
+    # To enable promscale in dkube. (true/false)
+    ENABLED={{ .Values.optional.promscale.enabled }}
+
+    BACKUP_ENABLED={{ .Values.optional.promscale.timescaledb.backup.enabled }}
+
+    # Timescale db details needs to provide if you are hosting 
+    # To use an external timescaledb. (true/false).
+    EXTERNAL={{ .Values.optional.promscale.timescaledb.external }}
+    DB={{ .Values.optional.promscale.timescaledb.db }}
+    USER={{ .Values.optional.promscale.timescaledb.user }}
+    HOST={{ .Values.optional.promscale.timescaledb.host }}
+    PASSWORD={{ .Values.optional.promscale.timescaledb.password }}
+    PORT={{ .Values.optional.promscale.timescaledb.port }}
+
+    # S3 details for backup only required if external is false and you want to use 
+    # custom s3 source for backup
+    # default: /timescale
```

Comment:
1. Can backup be done on demand, using a command as well. 
2. Someone may not have all these details when dkube is installed. How to update this information later?

Reply:
There is no provision for ondemand backup using k8s job but i think this tool can be used
Yes it can be done https://github.com/timescale/timescaledb-backup